certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+          "description": "Gets or Sets Amount",
+          "type": "number"
+        },
+        "loanPolicyLiability": {
+          "format": "double",
+          "description": "Gets or Sets LoanPolicyLiability",
+          "type": "number"
+        },
+        "lender": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "mortgageBroker": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "mortgageInsuranceCase": {
+          "description": "Gets or Sets MortgageInsuranceCase",
+          "type": "string"
+        },
+        "loanDetail": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.LoanDetail"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Financial": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "attributes": {
+          "description": "Gets or Sets Attributes",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "purpose": {
+          "description": "Gets or Sets Purpose",
+          "type": "string"
+        },
+        "transactionType": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TransactionType"
+        },
+        "purchasePrice": {
+          "format": "double",
+          "description": "Gets or Sets PurchasePrice",
+          "type": "number"
+        },
+        "earnestMoney": {
+          "format": "double",
+          "description": "Gets or Sets EarnestMoney",
+          "type": "number"
+        },
+        "downPayment": {
+          "format": "double",
+          "description": "Gets or Sets DownPayment",
+          "type": "number"
+        },
+        "ownerPolicyLiabilityOverride": {
+          "format": "double",
+          "description": "Gets or Sets OwnerPolicyLiabilityOverride",
+          "type": "number"
+        },
+        "loanPolicyLiabilityOverride": {
+          "format": "double",
+          "description": "Gets or Sets LoanPolicyLiabilityOverride",
+          "type": "number"
+        },
+        "cashOutAmount": {
+          "format": "double",
+          "description": "Gets or Sets CashOutAmount",
+          "type": "number"
+        },
+        "loans": {
+          "description": "Gets or Sets Loans",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Loan1"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.PropertyType": {
+      "format": "int32",
+      "description": "Gets or Sets PropertyType",
+      "enum": [
+        0,
+        1,
+        2,
+        3,
+        4,
+        5,
+        6,
+        7,
+        8,
+        9,
+        10
+      ],
+      "type": "integer"
+    },
+    "PubCS.TitleOrder.Api.Models.AddressType": {
+      "type": "string"
+    },
+    "PubCS.TitleOrder.Api.Models.Address": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.